Dockerfile.test

```diff
@@ -0,0 +1,5 @@
+FROM node:11.15.0
```

Comment:
FAR: see comment above. We could have one file only and use steps instead

Reply:
I'm not sure how to do this actually